src/qt/editor/welcomewidget.cpp

```diff
@@ -305,7 +323,7 @@ WelcomeWidget::WelcomeWidget(InviwoApplication* app, QWidget* parent)
 #endif
     };
 
-    if (getScreen()->size().width() < 1920) {
+    if (getScreen()->size().width() < 1200) {
         this->setSizes(QList<int>({1, 0}));
```

Comment:
is this still the right-most one after adding the other splitter on the left?

Reply:
Yes, there are two nested splitters now